### ISSUE TITLE
Agent not closing

### ISSUE THREAD

**Issue by grawsom** [sammachin/node-red-contrib-ngrok#46]
@sammachin 
Using latest ver. on Win11 and Raspberry Pi 3+ (NR 3.0.2)

Running Ngrok free version
If just taking the example (btw. should be updated), and opening/closing the tunnel, if you then do it on the next device you get an error. Worked fine on previous version.
It seems like when doing "off", you only close the tunnel, and not the agent.

![image](https://github.com/sammachin/node-red-contrib-ngrok/assets/6400532/7da6fe52-74fd-409a-8a4e-97afe9f6d333)

`Connect error: failed to connect session: Your account is limited to 1 simultaneous ngrok agent sessions.
You can run multiple tunnels on a single agent session using a configuration file.
To learn more, see https://ngrok.com/docs/secure-tunnels/ngrok-agent/reference/config/
You can view your current agent sessions in the dashboard:
https://dashboard.ngrok.com/tunnels/agents error_code: ERR_NGROK_108`

**Comment by vibraHome** [comment]
Same here. Any news?

**Comment by sammachin** [comment]
@grawsom Sorry I missed the notification for this issue, 

I can reproduce, taking a look at it now

**Comment by sammachin** [comment]
Seems to be a bug in the ngrok-javascript, waiting on a response on https://github.com/ngrok/ngrok-javascript/issues/148

**Comment by sammachin** [comment]
@grawsom don't know if you're still waiting on this but ngrok seem to have finally produced a fix (or someone in the community did!) 
Just waiting for them to release an updated package with the fix in and then I'll update the dependencies and push a new version for node-red

**Comment by grawsom** [comment]
Yes I have been following, and I will test when you are able to release.

**Comment by grawsom** [comment]
> [@grawsom](https://github.com/grawsom) don't know if you're still waiting on this but ngrok seem to have finally produced a fix (or someone in the community did!) Just waiting for them to release an updated package with the fix in and then I'll update the dependencies and push a new version for node-red

Can see it's 1.51 now,  when will you update the package?

**Comment by sammachin** [comment]
Hi @grawsom thanks for the nudge, I did ask on the ngrok thread about the release but I don't think I ever got a reply from them :(

I've updated the dependency and released a new node-red packge in 3.2.0, I've only got time to do some basic testing on this right now so please test and report back.

**Comment by grawsom** [comment]
Hi @sammachin I will do this one of the next days and come back with the result. :-)

**Comment by grawsom** [comment]
Hi @sammachin - and a Happy New Year
I have not been following up on this, because I made a different solution, but now I took a closer look into this, and it was still not closing the tunnel.
I made a fork - and did change the 1.5.1 to 1.7.0 - but that had no impact.
I then tried to manually change the "ngrok.js" file to see if I could get some results.
If I put "await ng.disconnect();" into this it works, but there goes about 20 sec. before the tunnel is closed.

Don't know if this helps.

`      if (['false', 'off', '0'].indexOf(String(msg.payload).toLowerCase()) != -1) {
        (async function () {
          try {
            if (node.listener) {
              await node.listener.close();
	      await ng.disconnect();
	      node.listener = null;`
